lib/chef/resource/chocolatey_source.rb

```diff
@@ -23,10 +23,17 @@ class ChocolateySource < Chef::Resource
       description "Use the chocolatey_source resource to add or remove Chocolatey sources."
       introduced "14.3"
 
-      property :source_name, String, name_property: true
-      property :source, String
-      property :bypass_proxy, [TrueClass, FalseClass], default: false
-      property :priority, Integer, default: 0
+      property :source_name, String, name_property: true,
+               description: "The name of the source to add. We'll use the resource's name if this isn't provided."
```

Comment:
Ideally we should avoid the use of `we` here.  How about "The resource's name will be used if this isn't provided." instead?